Frontend/shared/src/hooks/AuthContext/AuthContextType.ts

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+// Create an interface for AuthContext
+export interface AuthContextType {
+    isAuthenticated: boolean;
+    userPicture: string | null;
```

Comment:
Change to userPictureLink (other files might need similar change for this name change, too).